root/edit/cannot_vote.tt

```diff
@@ -0,0 +1,8 @@
+[% WRAPPER 'layout.tt' full_width=1 title=l('Error Voting Edit') %]
+    <h1>[% l('Error Voting Edit') %]</h1>
+    <p>
+        [% l('You’re currently not allowed to vote because an admin has revoked your privileges.
```

Comment:
Could we use the same message that's in root/layout.tt? I guess the only difference is `s/vote/edit or vote/`. But that seems perfectly fine and then won't have to be translated again.